modules/node-shared/src/main/scala/org/tessellation/node/shared/infrastructure/consensus/Consensus.scala

```diff
@@ -1,92 +1,14 @@
 package org.tessellation.node.shared.infrastructure.consensus
 
-import java.security.KeyPair
-
 import cats.effect.Async
-import cats.effect.std.{Random, Supervisor}
-import cats.kernel.Next
-import cats.syntax.flatMap._
-import cats.syntax.functor._
-import cats.{Eq, Order, Show}
-
-import scala.reflect.runtime.universe.TypeTag
 
-import org.tessellation.kryo.KryoSerializer
-import org.tessellation.node.shared.config.types.ConsensusConfig
-import org.tessellation.node.shared.domain.cluster.services.Session
-import org.tessellation.node.shared.domain.cluster.storage.ClusterStorage
 import org.tessellation.node.shared.domain.consensus.ConsensusFunctions
-import org.tessellation.node.shared.domain.gossip.Gossip
-import org.tessellation.node.shared.domain.node.NodeStorage
-import org.tessellation.node.shared.domain.seedlist.SeedlistEntry
 import org.tessellation.node.shared.infrastructure.gossip.RumorHandler
-import org.tessellation.node.shared.infrastructure.metrics.Metrics
-import org.tessellation.schema.peer.PeerId
-import org.tessellation.security.{Hasher, SecurityProvider}
-
-import io.circe.{Decoder, Encoder}
-import org.http4s.client.Client
-
-object Consensus {
-
-  def make[
```

Comment:
All this setup happens now in CurrencySnapshotConsensus and GlobalSnapshotConsensus classes. Additional layer of abstraction wasn't needed in my opinion and I also needed to pull some of the initializations out from here so in the end this `make` stopped being useful.